bool calibrate

---

string message

bool success
